docs/erp_epa_mapping_details/EPAMedicationDispense.html

```diff
@@ -10,16 +10,14 @@
 </head>
 
 <body>
-    <h2>Mapping: GEM_ERP_PR_MedicationDispense|1.2 in EPAMedicationDispense</h2>
-    <p>Version: 0.9</p>
-    <p>Stand: 08.05.2024</p>
+    <h2>Mapping: GEM_ERP_PR_MedicationDispense|1.4 in EPAMedicationDispense</h2>
```

Comment:
Ich möchte hier nicht zeigen, wie die "neue" MedicationDispense in der kommenden Version 1.4 in die ePA MedicationDispense gemappt wird, sondern die "aktuelle" MedicationDispense in der Version 1.2.
Diese Operation soll veranschaulichen, wie der E-Rezept-Fachdienst in der Übergangszeit mappt und aufzeigen, wie AVS von der "alten" in die "neue" Welt überführt werden.
Die Version 1.4 der ePA MedicationDispense ist ebenfalls interessant, hilft jedoch weder dem E-Rezept-Fachdienst-Team noch den AVS in der aktuellen Situation.

Reply:
Außerdem bitte noch die Version und den Stand der Information mit angeben!
`    <p>Version: 0.10</p>
    <p>Stand: 15.05.2024</p>`